.travis.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         - echo ${DOCKER_CLOUD_PWD} | docker login --username ${DOCKER_CLOUD_USER} --password-stdin
         - docker-compose -f src/main/docker/test-environment/docker-compose.yml up -d
         - ./gradlew --build-cache copyDependencies test jacocoTestReport dockerTag${OF_VERSION} || travis_terminate 1;
-        - sonar-scanner
+        - sonar-scanner || travis_terminate 1;
```

Comment:
No, reporting code smells or lack of coverage is fine but it will stop if the sonar-scanner failed with an exception, for example if it couldn't run because it didn't find the necessary code folders.
Without travis_terminate, in the case of an exception the build would continue and do cypress & karate, but in the end the build would still be failed because the sonar-scanner step failed.
My reasoning was that this way we don't have to wait the full 30 min before having to re-do a build anyway... but it's true that in this scenario if anything is also wrong with cypress for example you would only find out in the second build... What do you think ?

Reply:
In my opinion it is better to stop the build in case of exceptions on sonar-scanner.